### PR TITLE
Added background

### PR DESCRIPTION
**Pull request by Aayush-hub**
## What is the change?
Added background image.

## Related issue?
issue: #109 

## How was it tested?
Locally by contributor

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Capture

https://user-images.githubusercontent.com/65889104/108504013-479fee80-72db-11eb-91ec-cf3820756c17.mp4

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ if(process.env.NODE_ENV !== 'production'){
 }
 
 //Connecting to Mongo Database using ODM Mongoose-
-const URL = process.env.URL;
+const URL = "mongodb://localhost:27017/blogDB";
```
Comment (Pranav016) [inline app.js:32]: turn this back to `process.env.URL`
`"mongodb://localhost:27017/blogDB"` will only be used to run the website on a server locally on your machine.

**Review by Pranav016 — COMMENTED** [review]
Is the background responsive? Does it stay intact in mobile view ? Can you add a screen recording showing how background stays the same in different views. Asking this cause I feel adding image by itself without any media queries or other css might break in mobile view.

**Comment by Aayush-hub** [comment]
@Pranav016 Done with the changes!

**Review by Pranav016 — CHANGES_REQUESTED** [review]
### Shows this issue when I run the code on my machine. 
image

### Mobile view
image

### tablet view
image

**Comment by Aayush-hub** [comment]
@Pranav016 Sir, this was issue with website responsiveness not with the background image, though I am done with the changes and solved errors. Please review my PR.

**Comment by Pranav016** [comment]
> @Pranav016 Sir, this was issue with website responsiveness not with the background image, though I am done with the changes and solved errors. Please review my PR.

Yes, I agree there was some issue with responsiveness. A PR for making the site responsive got merged just now, you can try again by pulling that code.

**Review by Pranav016 — COMMENTED** [review]
It is working perfectly in mobile and tablet view but is still giving issue in desktop view.
image

Responsiveness is now fixed, a new PR for that got merged. You can try one more time now.

**Comment by Aayush-hub** [comment]
@Pranav016 Done with the changes. Please review my PR:)

**Review by Pranav016 — COMMENTED** [review]
Works fine now 👍  I saw a lot of media queries being put in the header.ejs under style tag clutters the file a little bit but, it's fine. We can work with it. The PR is ready to merge but the commit message `solved errors` that is used 2 times is incorrect. Please make a new PR with correct commit messages (try to be more specific with commit messages) or squash the current commits.

**Comment by Aayush-hub** [comment]
@Pranav016 Sure, making a new PR:)

**Pull request closed without merging**